python/mxnet/io/io.py

```diff
@@ -709,23 +709,29 @@ def _getdata(self, data_source, start=None, end=None):
 
     def _concat(self, first_data, second_data):
         """Helper function to concat two NDArrays."""
+        if (not first_data) and (not second_data):
+            return []
+        elif (not first_data) or (not second_data):
```

Comment:
The four lines 712 to 715 could be replaced with the line 714 to 715.

if not first_data or not second_data:
  return first_data if first_data else second_data

When first_data and second_data are both [], it will return [] too.

Reply:
done